README.md

```diff
@@ -65,7 +65,7 @@ Playing on Linux isn't supported currently, but a full-featured server is.
    cd build
    cmake .. -DSKYRIM_DIR="C:/Program Files (x86)/Steam/steamapps/common/Skyrim Special Edition"
    ```
-   For Linux users and users who don't have Skyrim SE installed:
+   For users who don't have Skyrim SE installed:
```

Comment:
Actually, Skyrim is pretty [playable on Linux](https://www.protondb.com/app/489830) :)

And, technically, one can pass path to game directory from Windows partition when dual-booting. So I don't think this note is necessary.

P.S. I can move this to a separate PR, but I don't think it really makes too much sense to be that much granular.

Reply:
This change is ok for me